web_src/css/repo/issue-list.css

```
@@ -82,6 +82,10 @@
   background-color: var(--color-secondary-dark-4);
 }
 
+#issue-list .flex-item-body > .flex-text-inline::before {
+  content: "|";
```

```suggestion
  content: "•";
```

This also matches GitHub's style.